src/modules/manual_control/manual_control_params.c

```diff
@@ -58,3 +58,14 @@ PARAM_DEFINE_INT32(MAN_ARM_GESTURE, 1);
  * @max 15
  */
 PARAM_DEFINE_FLOAT(MAN_KILL_GEST_T, -1.f);
+
+/**
+ * Enable arm/disarm stick gesture DJI style
```

Comment:
At a minimum please describe what it actually is and not "DJI".

Reply:
@dagar san.  I would change.

Reply:
DJI A3 is described.

Reply:
@muramura I think Daniel wanted to say that we shouldn't call this "DJI" at all and use something more descriptive instead. For example, what about "Dual stick gesture"? The parameter could be called `MAN_ARM_GES_TYPE` and you could select between "Single stick" and "Dual stick" gestures.